src/servant/tt_002/faerie.h

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include "common.h"
+#include <servant.h>
+
+typedef struct {
+    s16 animIndex;
+    s16 zPriorityFlag;
+} FaerieAnimIndex;
+
+// Ranked Lookup Table Macros
+#define RANKED_LOOKUP_TABLE(name) s32 name[]
```

Comment:
I would get another opinion before changing this but my opinion is that these add a step of indirection to what is a pretty common pattern in C. I think most C programmers would instantly understand `array[index * 2]` `array[index * 2 + 1]` etc. but have to look in the header and think harder to figure how how the macros get filled out

Reply:
I think that's a fine criticism.  Not coming from a C background, I didn't know that the array[index*2] array[index*2+1] pattern was a common pattern.  I'll revert that checkin.